### PR TITLE
fix for TTML localised whitespace preservation #3011

### PR DESCRIPTION
**Pull request by david-hm-morgan**

### CODE REVIEW

**Comment by david-hm-morgan** [comment]
@googlebot I signed it!


Sent from Outlook<http://aka.ms/weboutlook>

________________________________

File: lib/text/ttml_text_parser.js [hunk 1]
```diff
@@ -202,6 +202,12 @@ shaka.text.TtmlTextParser = class {
     }
     goog.asserts.assert(cueElement, 'cueElement should be non-null!');
 
+    let spaceStyle = null;
```
Comment (michellezhuogg) [inline lib/text/ttml_text_parser.js:205]: nit: let spaceStyle = cueElement.getAttribute...

**Review by michellezhuogg — COMMENTED** [review]
Hello David, could you please confirm that the text moves to the right in the demo with your change?

File: lib/text/ttml_text_parser.js (hunk 1 above)
Comment (david-hm-morgan) [reply lib/text/ttml_text_parser.js:205]: Yes after this solution moves as expected to the right.

Comment (david-hm-morgan) [reply lib/text/ttml_text_parser.js]: style addressed in a subsequent commit

**Review by joeyparrish — APPROVED** [review]
Looks good to me.  I'll run it through the build bot to test it.  Thanks for contributing!

**Comment by david-hm-morgan** [comment]
@joeyparrish I've just changed it to `const` with another commit, hopefully this will allow us to move forwards with #3011

**Comment by joeyparrish** [comment]
Sounds good, thanks!  Running the tests again now.

**Pull request merged**